lang/ruby/lib/avro/io.rb

```diff
@@ -292,7 +292,7 @@ def read_data(writers_schema, readers_schema, decoder)
         readers_schema.type_adapter.decode(datum)
       end
 
-      def read_fixed(writers_schema, readers_schema, decoder)
+      def read_fixed(writers_schema, _, decoder)
```

Comment:
For some of these methods where the arguments are trying to follow a standard pattern, would it be better to use `_readers_schema` instead of `_` for example? If the cop accepts that or can be configured to accept it.

Reply:
As a long-time rubyist I like the `_descriptive_name` approach, it makes reading the code to understand it much easier.